microsoft/testsuites/stress/stress_ng_suite.py

```diff
@@ -0,0 +1,165 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from pathlib import Path, PurePath
+from typing import Any, Dict, List, cast
+
+from lisa import (
+    Environment,
+    Logger,
+    Node,
+    RemoteNode,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    notifier,
+)
+from lisa.messages import SubTestMessage, TestStatus, create_test_result_message
+from lisa.testsuite import TestResult
+from lisa.tools import StressNg
+from lisa.util import SkippedException
+from lisa.util.process import Process
+
+
+@TestSuiteMetadata(
+    area="stress",
+    category="stress-ng",
+    description="""
+    A suite for running the various classes of stressors provided
+    by stress-ng.
+    """,
+)
+class StressNgTestSuite(TestSuite):
+    TIME_OUT = 3600
```

Comment:
default timeout is 3600 for each case

Reply:
Removed the timeout property for the tests. Using this to wait for the stress-ng tool to finish executing.